### PR TITLE
[synthetics] Allow spaces between variable brackets

### PR DESCRIPTION
**Pull request by lefebvree**
### What and why?

Match tests variables definition rule to allow whitespace between variable name and brackets.

### How?

Update variable template regex.

### Review checklist

- [x] Feature or bugfix MUST have appropriate tests (unit, integration)

### CODE REVIEW

File: src/commands/synthetics/utils.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ const PUBLIC_ID_REGEX = /^[\d\w]{3}-[\d\w]{3}-[\d\w]{3}$/
 const SUBDOMAIN_REGEX = /(.*?)\.(?=[^\/]*\..{2,5})/
 
 const template = (st: string, context: any): string =>
-  st.replace(/{{([A-Z_]+)}}/g, (match: string, p1: string) => (p1 in context ? context[p1] : match))
+  st.replace(/{{\s*([A-Z_]+)\s*}}/g, (match: string, p1: string) => (p1 in context ? context[p1] : match))
```
Comment (etnbrd) [inline src/commands/synthetics/utils.ts:39]: This RegEx seems already available in synthetics common: https://github.com/DataDog/synthetics-worker/blob/prod/common-public/src/variable.ts#L1
Though, it's way more permissive, so I am not sure we should use it here, but we might want to synchronize the two, no?

**Review by etnbrd — APPROVED** [review]
LGTM

File: src/commands/synthetics/utils.ts (hunk 1 above)
Comment (lefebvree) [reply src/commands/synthetics/utils.ts:39]: We don't use synthetics-common in datadog-ci but we should certainly synchronize them yes

**Review by m-rousse — APPROVED** [review]
Awesome :+1:

**Pull request merged**